test/helpers/cdn.js

```diff
@@ -17,22 +17,29 @@ describe('cdn helper', function () {
         theme_version_id: '123',
         theme_config_id: '3245',
     };
+    var themeSettings = {
+      cdn: {
+        customcdn: {
```

Comment:
I think this structure is unnecessary, the path is too nested. Can you change the code to be like this?

```
"settings": {
      cdn: {
        customcdn1: "cdn1.mystore.com",
        customcdn2: "cdn2.mystore.com"
    }
}
```

usage:
```
{{cdn "customcdn1:path/to/image.jpg"}}
{{cdn "customcdn2:path/to/image.jpg"}}
```

Also, if you can, add a test case with two custom cdn urls like the above